frontend/src/packages/orders/orders-api.ts

```diff
@@ -24,17 +26,19 @@ class OrdersApi extends HttpApi {
     super({ path: ApiPath.ORDERS, baseUrl, http, storage });
   }
 
-  public async getBusinessOrders(): Promise<OrderResponseDto[]> {
+  public async getBusinessOrders(
+    queryString: string,
+  ): Promise<OrdersListResponseDto> {
     const response = await this.load(
-      this.getFullEndpoint(OrdersApiPath.ROOT, {}),
+      this.getFullEndpoint(`${OrdersApiPath.BUSINESS}?${queryString}`, {}),
```

Comment:
You can do it like this, if you wish:
```suggestion
      this.getFullEndpoint(OrdersApiPath.BUSINESS, "?", queryString, {}),
```